service/src/main/java/uk/gov/hmcts/reform/fpl/model/C21Order.java

```diff
@@ -0,0 +1,14 @@
+package uk.gov.hmcts.reform.fpl.model;
+
+import lombok.Builder;
+import lombok.Data;
+import uk.gov.hmcts.reform.fpl.model.common.DocumentReference;
+
+@Data
+@Builder(toBuilder = true)
+// Temp c21 Order
```

Comment:
If this is the temp C21Order should we not name it that?

Reply:
Probably... there's not really a non-temporary C21. The C21 order is built during the event, then added to the c21 bundle, then deleted at the end of the event (all its information now stored in a bundle entry). Not sure if renaming refactor is necessary